src/Components/Top/TopViewItem.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import imgSample from './imgSample.svg'
+
+const TopViewItem = ({topViewList}) => {
+	const { name, text, click } = topViewList;
+
+	return (
+		<div className = "list-item">
+			<img className = "list-img" src = {imgSample} alt = "img"/>
```

Comment:
alt 같은 경우는 이미지 불러오기에 실패하였을 경우
대체 텍스트로 노출되는 친구이기 때문에 좀 더 의미있는 텍스트를 넣는 것을 추천드립니당
```suggestion
			<img className = "list-img" src = {imgSample} alt = `${topViewList.name} image`/>
```

Reply:
확인했습니다.
1. PR 데모이미지
2. Indent 통일
3. alt 메세지 
고쳐서 다시 올리겠습니다!

Reply:
@devbluesea

Reply:
@gyumnnii 
indent 문제가 저는 이상 없는데 push하면 계속 똑같아서 다시 만들어봤습니다.